packages/components/src/components/height/height.jsx

```diff
@@ -0,0 +1,39 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+class Height extends React.Component {
```

Comment:
Can we use a less verbose name for this `component`? `Height` seems a bit too general.

Reply:
Sure, how about `DynamicHeight`? anything specific in mind?

Reply:
as per slack discussion, `AutoHeightWrapper` it is.